.ddev/docker-compose.cypress.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  cypress:
+    image: cypress/included:12.6.0
+    container_name: ddev-${DDEV_SITENAME}-cypress
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: ${DDEV_APPROOT}
+    user: "1000:1000"
```

Comment:
Does it break without this? As per [this](https://github.com/tyler36/ddev-cypress/issues/26#issuecomment-1588302459), it is already taken care of.

Reply:
No, It doesn't break. I have removed the code.